src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -71,7 +72,7 @@ public PlayerBackpack(@Nonnull PlayerProfile profile, int id, int size) {
         this.cfg = profile.getConfig();
         this.size = size;
 
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue( CONFIG_PREFIX+ id + ".size", size);
```

Comment:
Please place that space after the variable.
```suggestion
        cfg.setValue(CONFIG_PREFIX + id + ".size", size);
```

Reply:
64c639ef8